src/components/status/readme.md

```diff
@@ -0,0 +1,47 @@
+The `sbb-status` is a component that has the purpose to show the user short messages to update the current status.
```

Comment:
As we don't provide any accessibility feature out of the box, I think we should provide the accessibility section in docs and especially describe how role status or similar can be used if needed.

Reply:
Comment resolved, discussion continues in https://github.com/lyne-design-system/lyne-components/pull/2262#discussion_r1429598643